astropy/time/tests/test_basic.py

```diff
@@ -852,7 +852,7 @@ def test_bad_input_subformat(self):
         """Non-existent input subformat"""
         with pytest.raises(ValueError):
             Time(
-                "2000-01-01 01:01", format="iso", scale="tai", in_subfmt="doesnt exist"
+                "2000-01-01 01:01", format="iso", scale="tai", in_subfmt="doesn't exist"
```

Comment:
This would not fit in the 88 character line length limit and would therefore force Ruff to split this over several lines. However, the following is short enough and also avoids using `'` in the string:
```suggestion
                "2000-01-01 01:01", format="iso", scale="tai", in_subfmt="invalid"
```
But I'm not going to make that change unless @taldcroft confirms that I should.

Reply:
Good point. "nonexistent" should also fit.

Reply:
For the record, I was not arguing for improving or extending the existing (previous) test, just would prefer not to weaken it by introducing a non-alphanumerical character to the string only for the sake of keeping a spellchecker happy about a variable _name_. But since @taldcroft has not commented since, I am willing to let his initial approval stand.

Reply:
Raising the question was reasonable enough, but if the sub-package maintainer doesn't think it should be addressed then this should be merged as is.